lms/envs/common.py

```diff
@@ -44,17 +43,15 @@
     ENTERPRISE_REPORTING_CONFIG_ADMIN_ROLE,
     ENTERPRISE_OPERATOR_ROLE
 )
-
-from openedx.core.constants import COURSE_KEY_REGEX, COURSE_KEY_PATTERN, COURSE_ID_PATTERN
+from lms.djangoapps.lms_xblock.mixin import LmsBlockMixin
 from openedx.core.djangoapps.theming.helpers_dirs import (
     get_themes_unchecked,
     get_theme_base_dirs_from_settings
 )
 from openedx.core.lib.derived import derived, derived_collection_entry
 from openedx.core.release import doc_version
-from xmodule.modulestore.modulestore_settings import update_module_store_settings
+from path import Path as path
 from xmodule.modulestore.edit_info import EditInfoMixin
-from lms.djangoapps.lms_xblock.mixin import LmsBlockMixin
```

Comment:
we usually avoid making changes in core files like these, even when it comes to isorting imports. Let's only make changes that are necessary in core files

Reply:
Makes sense. My bad.